doc/how_to_docs.rst

```diff
@@ -58,3 +65,9 @@ Then the following commands allow you to create the documentation locally:
 
 This will create html files in `pypsa-earth/doc/_build/html`.
 VScode provides a so called Liveserver extension such that the html file can be opened locally on your computer.
+
+.. note::
+    
+    To build the documentation, Windows users might need to replace the last command by:
+    
+        .../pypsa-earth/doc (pypsa-earth-docs) % ./make html
```

Comment:
Thanks a lot for thinking about platform specific details. It will be definitely very helpful

Just a double-check question: should it be really "./", not ".\" which is a usual path separator in Windows?
Could you also explain a bit on why does this fix help?

Reply:
After inspection, the original instruction `make html` now works well on the PowerShell of my Windows laptop. This was not the case in the past (that is why I looked for the `./make html` workaround). Some PowerShell configuration might have changed during this time that fixed it, but 

I can't tell what it is.

The command `.make html` does not work on my windows laptop. I get this error: [make_error.txt](https://github.com/pypsa-meets-earth/pypsa-earth/files/11440743/make_error.txt).

Since the standard `make html` seems to be working fine, shall we remove the Note box?

Reply:
@asolavi, thanks a lot for the investigation.

My feeling is that your findings may be a good support to get starting with sphinx. It seems from this [SO discussion](https://stackoverflow.com/questions/65471557/make-html-not-working-for-sphinx-documentation-in-windows-10) that the issues you experienced are quite common. 

As an idea, I'd suggest to keep the note changing its content to something more general. For example: 
1) Windows users can face some problems when trying to build docs locally depending on an OS version and terminal used;
2) a workaround definitely can be found;
3) we can suggest to look for a solution on SO (or something alike).

Although, I'm not sure if such instruction would be really meaningful and absolutely do not insist on it. Feel free to disagree :)

Reply:
Looks great, I think. Nice idea with a list to-be-continued :)